docs/lib/storage/fragments/flutter/download.md

```diff
@@ -10,3 +10,20 @@ try {
   print(e.toString());
 }
 ```
+
+### Generate a download URL
+
+You can also retrieve a URL for the object in storage:
+
+```dart
+
+try {
+  GetUrlResult result =
+    await Amplify.Storage.getUrl(key: "myKey", options: options);
```

Comment:
We should not pass an options param in the API call if we didn't create the object itself.